sdv_model/proto/seats.proto

```diff
@@ -1,144 +0,0 @@
-// Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
-//
-// This program and the accompanying materials are made available under the
-// terms of the Apache License, Version 2.0 which is available at
-// https://www.apache.org/licenses/LICENSE-2.0.
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-// License for the specific language governing permissions and limitations
-// under the License.
-//
```

Comment:
is it correct that the proto-files are removed?

Reply:
That needs to be clarified, because in the ticket nothing is written about it